doc/DevelopersGuide/scheme_template.meta

```diff
@@ -0,0 +1,78 @@
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+! CCPP-compliant physics scheme template metadata
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!
+! Metadata rules:
+!
+! - metadata files (.meta) are in XML format and contain metada for one or more CCPP entrypoint schemes. 
```

Comment:
Please implement the comments and corrections from CompliantPhysicsParams.rst in this file as they all apply (e.g., the files are not in XML format).

Reply:
Done.